sympy/unify/core.py

```diff
@@ -35,7 +35,7 @@ def __hash__(self):
         return hash((type(self), self.op, self.args))
 
     def __str__(self):
-        return "%s[%s]" % (str(self.op), ', '.join(map(str, self.args)))
+        return f"{self.op}[{', '.join(map(str, self.args))}]"
```

Comment:
Suggestion: wherever you have {.*join.*} consider defining the joined object (like is done in some places) and then using that object in the f-string, e.g. `args = ','.join(map(str, self.args))` then `f"{self.op}[{args}]"`.